avm/res/sql/server/encryption-protector/main.bicep

```
@@ -8,10 +8,10 @@ param sqlServerName string
 @description('Required. The name of the server key.')
 param serverKeyName string
 
-@description('Optional. Key auto rotation opt-in.')
-param autoRotationEnabled bool = false
+@description('Optional. Key auto rotation opt-in flag. Either true or false.')
```

```suggestion
@description('Optional. Key auto rotation opt-in flag.')
```
I guess that should be a given 😉

---

Agree. My principle was always to take the original description of the RP (in this case [here](https://learn.microsoft.com/en-us/azure/templates/microsoft.sql/servers/encryptionprotector?pivots=deployment-language-bicep#encryptionprotectorproperties)), but we can simplify those messages in similar cases.